### ISSUE TITLE
Modify minority rating (just opportunity)

### ISSUE THREAD

**Issue by alecramsay** [rdatools/rdabase#51]
This issue tracks my work to explore minority ratings for the plans in ensembles.

With the first (current) version of the v2 trade-offs site, we saw this weird modality in the scatter plots involving minority ratings (see next image). 
![NC20C_proportionality_minority_scatter](https://github.com/user-attachments/assets/629be6c9-da5d-466b-b176-529556d46e06)

I hypothesized that it was due to the non-linear combination of a rating for minority opportunity districts and a rating for minority _coalition_ districts, so I created an alternate opportunity-district-only minority rating and re-scored the NC ensemble. I expected to see a continuous 2D ratings cloud. Much to my surprise, I did not (see next image).
![NC20C_proportionality_minority_scatter](https://github.com/user-attachments/assets/781cbdda-86b3-45b3-9822-12a49abdd62b)

This revised rating is a function of the ratio of number of opportunity (fractional) divided by the proportional number given the statewide demographic share. I expected this simple ratio to change smoothly and more-or-less continuously, but the plot shows an unexpected chunkiness, a banding.

Let's discuss.

**Comment by alecramsay** [comment]
I ran this abbreviated and modified pipeline on MD, so I'd have a second state to look at thinking that the modality in the NC alternate opportunity-district-only minority ratings might reflect some characteristic of the NC demographic geographic and be different in another state ... but no: MD has the same three-band kind of ratings (see image). 
![MD20C_minority_compactness_scatter](https://github.com/user-attachments/assets/db619e1c-c126-4920-8962-e4de846df96e)
I need to noodle on this and how the metric itself might be complicit.

**Comment by alecramsay** [comment]
Actually, as I look at the MD scatter plot more carefully, it's actually got four bands ... but there's a clear banding with quantum jumps in between them.

Right now, I don't see how (what I think is a continuous function) could produce this result, so my suspicion is that this reflects the recombination process and that every now and again an individual recombination makes a "leap" in terms of the demographic stats which then gets built on (more plans) until another "leap" to another plateau. 

Wild, if true. But would also beg the question: How does this not happen for any other dimensions, in particular proportionality and competitiveness. Those three dimensions are, I think, highly correlated. I'm puzzled ...

**Comment by alecramsay** [comment]
I re-worked an alt. minority rating to ignore coalition districts and just use the unshifted seat probability ... and neither NC nor MD had an positive minority ratings. Either we need the shifting to "get in the money" -or- I have a bug.

**Comment by alecramsay** [comment]
The discontinuity in minority ratings was caused by clipping minority VAP %'s below 37%.

I've added an alternate minority rating that doesn't clip in this way and verified the results on NC, MD, and SC. So, the minority ratings in our ensembles are (slightly) different than the ratings for Notable Maps from DRA.

I haven't deployed updated scatter plots yet, because the scatter plots use a lot of information from the previous, now out-of-date pipeline runs. Once I resolve whether Moon's approach to approximating the frontiers (#44) works, I'll update the pipeline and regenerate all the artifacts consistently.
